apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -164,6 +164,27 @@
     ST =:= ?status_connecting; ST =:= ?status_connected; ST =:= ?status_disconnected
 ).
 
+%% If owner_id exists, log owner ID, otherwise log resource_id
+-define(LOG(LEVEL, FIELDS, DATA),
```

Comment:
Maybe wrap the body into a function or use a function? Otherwise, we will get "variable unsafe" if we use this macro twice in a scope.